infrastructure/kubernetes/mezo-production/monitoring/ingress.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: monitoring
+  namespace: monitoring
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: "mezo-production-monitoring-hub-external-ip"
+    ingress.gcp.kubernetes.io/pre-shared-cert: "mezo-production-monitoring-hub-ssl-certificate"
+spec:
+  rules:
```

Let's add a default backend so when we hit the root DNS, we get grafana (same for mezo-staging):
```suggestion
  defaultBackend:
    service:
      name: grafana
      port:
        number: 3000
```

---

addressed in: 3c17aff8b9ad03853b3bd7f1d4c145a5dc3810a2